src/App.js

```diff
@@ -3,7 +3,6 @@ import Calculator from './components/Calculator';
 // eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [ ] Please, consider removing all linter disables. Let the linter makes it is direct hob - help to find and fix bugs in the early steps of building the app 👍